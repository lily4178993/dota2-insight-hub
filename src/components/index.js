export { default as Header } from './Header/Header';
export { default as Card } from './Card/Card';
export { default as Filter } from './Filter/Filter';
export { default as Footer } from './Footer/Footer';
export { default as HeroInfo } from './DesignPresentations/HeroInfo';
export { default as ItemPresentation } from './DesignPresentations/ItemPresentation';
export { default as MatchInfo } from './DesignPresentations/MatchInfo';
export { default as ProPlayerProfile } from './DesignPresentations/ProPlayerProfile';
export { default as GridViewToggle } from './ux/GridViewToggle';
export { default as HeroCarousel } from './ux/HeroCarousel';
export { default as HeroGrid } from './ux/HeroGrid';
export { default as HeroList } from './ux/HeroList';
export { default as SearchPanel } from './ux/SearchPanel';
