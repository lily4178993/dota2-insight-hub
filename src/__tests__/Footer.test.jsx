import React from 'react';
import { render } from '@testing-library/react';
import { Footer } from '../components';

test('Footer component matches snapshot', () => {
  const { asFragment } = render(<Footer />);
  expect(asFragment()).toMatchSnapshot();
});
