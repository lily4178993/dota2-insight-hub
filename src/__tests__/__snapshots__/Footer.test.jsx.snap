// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    <div
      class="footer-container"
    >
      <div
        class="footer-content"
      >
        <p>
          Â© 2023 Dota2 Insight Hub
        </p>
        <ul
          class="footer-links"
        >
          <li>
            <a
              href="/"
            >
              Home
            </a>
          </li>
          <li>
            <a
              href="/aboutme"
            >
              About
            </a>
          </li>
          <li>
            <a
              href="/references"
            >
              References
            </a>
          </li>
          <li>
            <a
              href="#contact"
            >
              Contact
            </a>
          </li>
        </ul>
      </div>
    </div>
  </footer>
</DocumentFragment>
`;
